### ISSUE TITLE
When player first starts, if a captions track is selected by default, screen reader announces "selected" even though the captions menu does not have focus

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/video.js#5172]
## Description
Using, for example, the descriptions.html.example file in the sandbox, after a screen reader user (JAWS + IE11) clicks the Big Play Button (BPB), the screen reader announces "selected"

This is because the menu items are marked as `aria-live`, and when playback starts the default track has hidden text of ", selected" added.

#### videojs
v7.0
#### browsers
Observed with JAWS 2018 + IE11
